docs/pokemonCreation.md

```diff
@@ -0,0 +1,72 @@
+## Pokemon Creation
+
+The two main files that handle Pokemon creation are [`Equations.ts`](../src/components/Equations.ts) and [`Utilities.ts`](../src/components/Utilities.ts).
+`Equations.ts` includes functions for choosing random wild Pokemon and for creating the actual Pokemon object.
+`Utilities.ts` facilitates the creating of a wild Pokemon by preparing parameters then calling `Equations.ts` creation of the Pokemon object.
+
+A newly created Pokemon's member types can be found in [`Battles.ts`](../src/components/Battles.ts) under the `IPokemon` interface.
+
+### Equations
+
+`Equations.ts` contains the functions necessary for choosing a random Pokemon (`chooseRandomWildPokemon`), creating a new Pokemon (`newPokemon`), choosing that new Pokemon's statistics (`newPokemonStatistics`), and choosing a new Pokemon's IVs and EVs (`newPokemonIVs` and `newPokemonEvs`).
+
+### Utilities
+
+`Utilities.ts` contains the function necessary for calling `newPokemon` in `Equations.ts` (`createPokemon`), which uses the chosen random Pokemon, from `Equations.ts` `chooseRandomWildPokemon`, to get the get the level of the chosen Pokemon and title which it then feeds to `newPokemon`.
+
+### Unit Test and Trainer Battles
+
+Pass into the `newPokemon` function from `Equations.ts` an object of type `INewPokemon`.
+`INewPokemon` holds information on the Pokemon being created and eventually helps spit out a fully created Pokemon.
+
+```javascript
+const pokemonTitle: string[] = "CHARMANDER".split("");
+const pokemonLevel: number = 10;
+const pokemon: IPokemon = FSP.equations.newPokemon({
+    level: pokemonLevel,
+    title: pokemonTitle
+});
+const pokemon: IPokemon = FSP.equations.newPokemon({
+    item: "Potion".split(""),
+    level: pokemonLevel,
+    moves: [{
+        title: "Ember",
+        remaining: FSP.constants.moves.Ember.pp,
+        uses: FSP.constants.moves.Ember.pp
+        }],
+    title: pokemonTitle
+});
+```
+
+### Wild Pokemon Encounter
+
+The steps for creating a Pokemon in a wild Pokemon encounter are:
+1. The `chooseWildPokemonForBattle` function is run.
+    * The grass `IMap` and `IArea` determine which wild Pokemon can be encountered by looking at the current location which is an object labeled `areas` that includes an object `wildPokemon` that holds what Pokemon can be found along with appearance rates.
```

Comment:
As before: there's no need to put this in a single bulleted list item. Just keep it as part of the same `1.`.